docs/source/release-notes.rst

```diff
@@ -3,6 +3,44 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+.. _release-notes-v0.7.0:
+
+FiftyOne 0.7.0
+--------------
+*Released December 21, 2020*
+
+App
+^^^
+- Added web browser support, which is now the default settting
+- Added :ref:`IPython notebook support <notebooks>`, e.g. Jupyter and Google
+  Colab
```

Comment:
@benjaminpkane FYI I removed the VS Code mention here; lmk if that was wrong to do